flink-table-store-core/src/main/java/org/apache/flink/table/store/sink/SinkRecord.java

```diff
@@ -31,20 +31,15 @@
 
     private final int bucket;
 
-    private final RowKind rowKind;
-
     private final BinaryRowData key;
 
     private final RowData row;
 
-    public SinkRecord(
-            BinaryRowData partition, int bucket, RowKind rowKind, BinaryRowData key, RowData row) {
+    public SinkRecord(BinaryRowData partition, int bucket, BinaryRowData key, RowData row) {
```

Comment:
Nit: If the`rowKind` is removed from `SinkRecord`, we'd better update the comment as well.
```
/** A sink record contains key, value and partition, bucket information. */
```